lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -509,6 +517,23 @@ fromCardanoBlock gp = \case
     BlockAlonzo blk ->
         fst $ fromAlonzoBlock gp blk
 
+numberOfTransactionsInBlock :: CardanoBlock StandardCrypto -> (Int, (Quantity "block" Word32, O.SlotNo))
+numberOfTransactionsInBlock = \case 
+
+    BlockShelley shb -> transactions shb 
+    BlockAllegra shb -> transactions shb 
+    BlockMary shb -> transactions shb 
+    BlockAlonzo shb -> transactionsAlonzo shb 
+    BlockByron byb -> transactionsByron byb 
+    where 
+        transactions (ShelleyBlock (SL.Block (SL.BHeader header _) (SL.TxSeq txs')) _) 
+            = (length txs', (fromBlockNo $ SL.bheaderBlockNo header, SL.bheaderSlotNo header))
+        transactionsAlonzo (ShelleyBlock (SL.Block (SL.BHeader header _) (Alonzo.TxSeq txs')) _) 
+            = (length txs' , (fromBlockNo $ SL.bheaderBlockNo header, SL.bheaderSlotNo header))
+        transactionsByron blk = (, (fromBlockNo $ O.blockNo blk, O.blockSlot blk)) $ case byronBlockRaw blk of
```

Comment:
🤼 "80 characters — no 100 — no 80"

```suggestion
        transactionsByron blk = (, (fromBlockNo $ O.blockNo blk, O.blockSlot blk)) $
            case byronBlockRaw blk of
```

"But that increases the indentation of the next lines — 80 characters" 🤼🤼😂